dataactbroker/handlers/accountHandler.py

```diff
@@ -660,52 +660,54 @@ def listUsersWithStatus(self):
             userInfo.append(thisInfo)
         return JsonResponse.create(StatusCode.OK,{"users":userInfo})
 
-    def listSubmissionsByCurrentUserAgency(self):
-        """ List all submission IDs associated with the current user's agency """
+    def list_submissions(self, page, limit, certified):
+        """ List submission based on current page and amount to display. If provided, filter based on
+        certification status """
+
+        # convert params and type check
+        try:
+            page = int(page) if page is not None else 1
+        except:
+            raise ValueError("Incorrect type specified for 'page'. Please enter a positive number.")
+
+        try:
+            limit = int(limit) if limit is not None else 5
+        except:
+            raise ValueError("Incorrect type specified for 'limit'. Please enter a positive number.")
+
+        if certified is not None:
+            certified = certified.lower()
+        # If certified is none, get all submissions without filtering
+        if certified is not None and certified not in ['true', 'false']:
+            raise ValueError("Incorrect value specified for the 'certified' parameter")
```

Comment:
I originally had it in the route function and I think we should move it back there. That way any subsequent method will assume the correct type and function normally.

I didn't convert `certified` because sqlalchemy doesn't care as long as the string matches True or False. But I can convert if that's best practice.